### PR TITLE
Test glitches

### PR DESCRIPTION
**Pull request by ellert**
# This Pull request:

## Changes or fixes:

Fix some issues with test dound in 6.30.00

## Checklist:

- [x] tested changes locally
- [ ] updated the docs (if necessary)

### CODE REVIEW

File: bindings/pyroot/pythonizations/test/rtensor.py
Comment (guitargeek) [inline bindings/pyroot/pythonizations/test/rtensor.py:35]: What if we scrap this extra check to alter the `cpptypes`, and directly define them to be what they are?

```c++
    cpptypes = ["int32_t", "int64_t", "uint32_t", "uint64_t", "float", "double"]
```
At least this is what I understand from the numpy docs:
https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.int64

File: bindings/pyroot/pythonizations/test/rdataframe_makenumpy.py
Comment (guitargeek) [inline bindings/pyroot/pythonizations/test/rdataframe_makenumpy.py:150]: Similar to the other test, should we maybe better use `df.Take['int64_t']` further down to be consistent with NumPy?

**Review by guitargeek — CHANGES_REQUESTED** [review]
Thanks @ellert. I just have some suggestions about the integer types.

File: tutorials/CMakeLists.txt
Comment (ellert) [reply tutorials/CMakeLists.txt:78]: There is no request for changes here, so I will resolve this.

**Comment by guitargeek** [comment]
Hi @ellert, sorry that my suggestions made things even worse and the tests don't pass anymore.

I'll revive this PR myself, updating the commits as necessary before merging and backporting.

File: bindings/pyroot/pythonizations/test/rtensor.py
Comment (guitargeek) [inline bindings/pyroot/pythonizations/test/rtensor.py:23]: I don't understand why the obvious mapping of `int64` to `int64_t` doesn't work on macOS, but on Windows and Linux it works fine.

Therefore, I'm treating macOS/Darwin as the special case now.

**Review by guitargeek — APPROVED** [review]
Thanks for the initiative! I'll merge this once the CI is green (the backport is also ready).

Please let us know with a GitHub issue or a new PR if there should be still failures on Linux 32 bit in your case.

**Pull request merged**
